### ISSUE TITLE
Low rank requirements for commands

### ISSUE THREAD

**Issue by RedRafe** [Factorio-Biter-Battles/Factorio-Biter-Battles#475]
https://github.com/Factorio-Biter-Battles/Factorio-Biter-Battles/blob/c3e2b3617ab3c21eaa95531ee3c17c34ed07c38a/utils/profiler.lua#L25-L41

I think the level required to run the profiler should be raised to `admin` at least. Often times during the captains game every new joiner gets insta-trusted for the session (& since the adding to whitelist is not working 100% correctly either). Players could accidentally or even intentionally start profiling session during games

**Comment by RedRafe** [comment]
I really liked the idea of a "super trusted" rank. Mainly because trusted/untrusted unlocks the blueprint/Decon planner that's all that player's really want when playing (obviously). Still, I don't see why a trusted player would ever need to run a profiler during games & risk to desync everyone

**Comment by clifffrey** [comment]
Yeah, as I said, I'm a-ok with changing this to be admin-only.  This would be an easy PR for someone to make.

**Comment by seghue** [comment]
Running locally, a player cant run `/startProfiler` unless the game is launched with `--enable-unsafe-lua-debug-api`.
So it's currently 'safe' assuming the server isn't being launched with this option, and it is not running 1.1.106 or earlier(?).
The file has been revised quite a bit since this issue was added, including a shorter auto-end if called by a non-admin

**Comment by cl33per** [comment]
Since this issue is over two years old, is this still relevant? 
Looks like the function has been modified around this:
https://github.com/Factorio-Biter-Battles/Factorio-Biter-Battles/blob/44ac61857d897077b550d3db52e23151bc0e7a34/utils/profiler.lua#L19-L88
